docs/examples.rst

```diff
@@ -1,4 +1,110 @@
+.. _examples:
+
 Examples
 ========
 
-Coming soon.
+Basics
+------
+Provided in the source are several examples that can help you to 
+get started using MetSim. They are located in the ``examples``
+directory.  We will look at the ``example_nc.conf`` file.  Its 
+contents are:
+
+.. code-block:: ini
```

Comment:
I wonder if it is possible to include the text from the file here rather than maintaining two copies of it...